.gitlint

```diff
@@ -106,11 +109,16 @@ regex=^.*
 # Use 'all' to ignore all rules
 # ignore=T1,body-min-length
 
-# [ignore-body-lines]
+[ignore-body-lines]
 # Ignore certain lines in a commit body that match a regex.
 # E.g. Ignore all lines that start with 'Co-Authored-By'
 # regex=^Co-Authored-By
 
+# ignore lines that are "footnotes", that start like `[1]: ` or `[2]: ` and so on
+# this will make it easy to put long urls in commit messages without
+# triggering gitlint body rules
+regex=^\[[0-9]+\]: +
```

Comment:
I would suggest to leave colon(`:`) out as a requirement in the regex pattern for footnotes.

Reply:
OK, it is now optional.